fix MUCX

brossignol opened the pull request:
Hello, I propose a fix to a MUCX error and a small improvement:

* fix `MUCX.compute` returned indices shifted by +1, they should not (at least I don't get why they should be shifted).
* fix a typo in assert argument, the solver name is `solver` (`solver_name` variable doesn't exist)
* Propose a fix to #162 by adding `formula.weighed()` directly inside the `MUCX.__init__`.

alexeyignatiev commented:
Thank you for the contribution, @brossignol. I have the following comments:
1. Clause indices reported by MUSx in the command line are correct. They are meant to start from 1 and the list of clauses reported is zero-terminated. (This is in line with how the other tools report their solutions.) So this change should be removed.
2. If we want to make sure `CNF` formulas are supported, we should do the same for `CNFPlus` formulas. Please add that too (should be simple).
Everything else looks good. Again, thank you!

Best,
Alexey

brossignol commented:
Hello, I updated the fix.

* Reverted the shift to previous. It makes sense if other tools are using 1-based array.
* CNFPlus are already supported because the CNFPlus class inherits from CNF class (`isinstance(CNFPlus, CNF) >> True`).

alexeyignatiev commented:
OK, thanks!
